api-gateway/src/main/java/com/petclinic/bffapigateway/domainclientlayer/CustomersServiceClient.java

```diff
@@ -119,4 +113,47 @@ public Mono<OwnerDetails> deleteOwner (final long ownerId) {
                 .bodyToMono(OwnerDetails.class);
     }
 
+    public Mono<String> setPhotoOwner(PhotoDetails file, int id){
+        return webClientBuilder.build().post()
+                .uri(customersServiceUrl +"/photo/" + id)
+                .body(just(file), PhotoDetails.class)
+                .retrieve().bodyToMono(String.class);
+    }
+
+    public Mono<PhotoDetails> getPhotoOwner(int id){
+        return webClientBuilder.build().get()
+                .uri(customersServiceUrl +"/photo/" + id)
+                .retrieve()
+                .bodyToMono(PhotoDetails.class);
+    }
+
+    public Mono<Void> deletePhotoOwner (int photoId) {
+        return webClientBuilder.build().delete()
+                .uri(customersServiceUrl + "/photo/" + photoId)
+                .retrieve()
+                .bodyToMono(Void.class);
+    }
+
+
+
+    public Mono<String> setPhotoPet(int ownerId, PhotoDetails file, int id){
+        return webClientBuilder.build().post()
+                .uri(customersServiceUrl + ownerId + "/pets/photo/" + id)
```

Comment:
Since we are duplicating this string `"/pets/photo/"` maybe we can refactor it into its own variable